core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -432,7 +441,8 @@ protected void configurePipeline(ChannelHandlerContext ctx, String protocol) thr
         private void addHttp2Handlers(ChannelHandlerContext ctx) {
             final ChannelPipeline p = ctx.pipeline();
             p.addLast(newHttp2ConnectionHandler(p, SCHEME_HTTPS));
-            p.addLast(new HttpServerHandler(config, gracefulShutdownSupport, null, H2, proxiedAddresses));
+            p.addLast(new HttpServerHandler(configHolder,
+                    gracefulShutdownSupport, null, H2, proxiedAddresses));
```

Comment:
nit: indent?

Reply:
I used intellij to reformat these lines of code. I have configured my ide to use Line corps settings.jar. Not sure what is the problem.